site/training/program/training-faq.qmd

```diff
@@ -170,7 +170,8 @@ Here's what we have been asked about {{< var vm.product >}} during training sess
 :::
 
 ::: {.w-40-ns .nb3 .pl2}
-[{{< fa graduation-cap >}} training](https://docs.validmind.ai/training/developer-fundamentals/developer-fundamentals.html){.button-small target="blank"} [{{< fa book-open >}} docs](https://docs.validmind.ai/guide/model-inventory/register-models-in-inventory.html){.button-small target="blank"}
+[{{< fa graduation-cap >}} training](#){.button-small}
```

Comment:
Just leaving the link part out from a Markdown link still leaves the button, and so you need to remove the actual link:


```suggestion

```

Here's what I currently see:

![Screenshot 2025-06-13 at 1 19 31 pm](https://github.com/user-attachments/assets/2714ddf8-196c-4cdc-bcd3-3e6f379e5d29)